settings.php

```diff
@@ -0,0 +1,241 @@
+<?php
+/**
+ * Plugin Settings
+ *
+ * @package rest-api-guard
+ */
+
+namespace Alley\WP\REST_API_Guard;
+
+add_action( 'admin_menu', __NAMESPACE__ . '\on_admin_menu' );
+add_action( 'admin_init', __NAMESPACE__ . '\on_admin_init' );
+
+/**
+ * Slug for the settings.
+ *
+ * @var string
+ */
+const SETTINGS_KEY = 'rest_api_guard';
+
+/**
+ * Register the Admin Settings page.
+ */
+function on_admin_menu() {
+	add_options_page(
+		__( 'REST API Guard', 'rest-api-guard' ),
+		__( 'REST API Guard', 'rest-api-guard' ),
+		'manage_options',
+		SETTINGS_KEY,
+		__NAMESPACE__ . '\render_admin_page',
+	);
+}
+
+/**
+ * Render the admin settings.
+ */
+function render_admin_page() {
+	?>
+	<div class="wrap">
+		<h2>
+			<?php esc_html_e( 'REST API Guard', 'rest-api-guard' ); ?>
+		</h2>
+		<!-- <p></p> -->
+		<?php settings_errors(); ?>
+
+		<form method="post" action="options.php">
+			<?php
+				settings_fields( SETTINGS_KEY );
+				do_settings_sections( SETTINGS_KEY );
+				submit_button();
+			?>
+		</form>
+	</div>
+	<?php
+}
+
+/**
+ * Register the admin settings.
+ */
+function on_admin_init() {
+	register_setting(
+		SETTINGS_KEY,
+		SETTINGS_KEY,
+		[
+			'sanitize_callback' => __NAMESPACE__ . '\rest_api_guard_sanitize_settings',
```

Comment:
Since the file is namespaced already, recommend removing namespacing from function name:
```suggestion
			'sanitize_callback' => __NAMESPACE__ . '\sanitize_settings',
```